dev/doctl.ts

```diff
@@ -0,0 +1,513 @@
+const DigitalOceanCTLCompletionSpec: Fig.Spec = {
+  name: "doctl",
+  description: "The official DigitalOcean command line interface (CLI)",
+  subcommands: [
+    {
+      name: "completion",
+      description:
+        "Configure your terminal's shell so that doctl commands autocomplete when you press the TAB key",
+      additionalSuggestions: ["bash", "zsh", "fish"],
```

Comment:
We should use an `arg` along with `suggestion` here:
```suggestion
      args: {
        name: "shell",
        suggestions: ["bash", "zsh", "fish"],
      },
```